package.json

```diff
@@ -46,6 +46,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.61",
     "@types/luxon": "^3.4.2",
+    "dayjs": "^1.11.13",
```

Comment:
IIRC, sadly, backstage has preferences and standards about what date libraries to use. I'll see if I can hunt down the docs. Prob not a big deal to use dayjs, but might be worthwhile to participate in their standards to be a good open-source citizen.

Reply:
I peeked in `yarn.lock` and saw that Material UI has dependencies on [date-io](https://github.com/dmtrKovalenko/date-io) for date pickers, which apparently is an abstraction layer on top of other date libraries?? Looks like there's specifically a `@date-io/date-fns` dependency, so maybe `date-fns` is the way to go? I'm puzzled, because I don't actually see a transitive dependency for `date-fns` itself 🤔

Reply:
Hmm but then I'm also seeing imports in a couple of our plugin components from [luxon](https://github.com/moment/luxon). Maybe that's the one we want.

Reply:
Yep, that was it. Here's an [ADR](https://backstage.io/docs/architecture-decisions/adrs-adr010) saying to use Luxon. Cool, I'll switch to that then!

Reply:
Came here to link to that haha yeah lets use luxon